apps/a11y-tests/src/tests/__snapshots__/ComponentExamples.test.tsx.snap

```diff
@@ -1009,7 +1009,7 @@ Array [
       Object {
         "logicalLocations": Array [
           Object {
-            "fullyQualifiedName": "button[aria-label=\\"June\\\\ 2017\\"]",
+            "fullyQualifiedName": "button[aria-label=\\"May\\\\ 2017\\"]",
```

Comment:
Hmmm… what is with all this Month changes 🤔

Reply:
K. there is a separate discussion about this so it’s not something your changes are bringing in.